docs/about/index.rst

```diff
@@ -17,13 +42,118 @@ Code of conduct
+
+   - **Improve existing notebooks and create new notebooks** that
+     `make fairness issues concrete https://fairlearn.github.io/contributor_guide/contributing_example_notebooks.html>`_
+     while demonstrating how to apply Fairlearn. To accelerate this process, we are experimenting with
+     :ref:`weekly sessions <notebook_deep_dive>` where people can discuss
+     individual example notebooks in detail.
+     
+   - **Improve the project documentation**: critique the current content, expand user guides,
+     improve writing and examples, port the current Python documentation to the numpydoc format. 
+     Convert existing example notebooks from `.ipynb` into `.py` files that
+     `render properly on the website <https://fairlearn.github.io/auto_examples/notebooks/index.html>`_,
+     while still being downloadable as `.py` or `.ipynb` files.
+
+   - **Improve the usability, relevance, and look of Fairlearn website** with the audience of practitioners in mind.
+     The current `website proposal <https://github.com/fairlearn/fairlearn-proposals/blob/master/documentation/STRUCTURE.md>`_
+     outlines the planned efforts, some of which have been completed. The plan itself is open for discussion.
+
+   - **Improve the usability and relevance of fairness metrics** by
+     `critiquing and improving the current metrics API <https://github.com/fairlearn/fairlearn-proposals/issues/12`_,
+     suggesting new metrics motivated by concrete use cases, and implementing the new metrics.
+   
+   - **Move towards compatibility with scikit-learn**: identify non-compatible aspects, improve code
+     towards compatibility.
+     While we aim for compatibility there may be aspects that are too
+     restricting for Fairlearn, so this may need to be evaluated on a
+     case-by-case basis.
+
+#. *Grow and nurture a diverse community of contributors*
+   
+   - **Reach out** with feedback on what is working and what
+     is not working; suggest how to improve things; point out where
+     documentation, our processes or any other aspect of the projects create
+     barriers of entry.
+
+   - **Participate** in our :ref:`monthly developer calls <developer_calls>`.
+     We are experimenting with adding :ref:`deep dive sessions <notebook_deep_dive>` around sociotechnical
+     aspects of notebooks.
+     We also work with universities to engage student contributors
+     through course projects and are also open to other forms of collaboration&mdash;let us know if you
+     are interested.
+
+   - **Improve the Fairlearn website and documentation** with the audience of contributors in mind.
+   
+   - **Add tests and improve testing infrastructure.**
+     
+#. *Create metrics, assessment tools, and algorithms to cover more complex ML tasks*
+
+   - **Create notebooks and use cases** that deal with concrete fairness issues in complex ML tasks including
+     ranking, counterfactual estimation, text, computer vision, and speech.
```

Comment:
would be nice to link to the guide to write notebooks

Reply:
done.